app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts

```diff
@@ -136,28 +137,31 @@ export class LocationInputComponent implements OnInit {
   }
 
   get latitudeField(): FormConfigField {
-    if (this.object.isSpaceGeom) {
+    if (this.object.isSpaceGeom && this.object.isSpaceGeom.embeddedFields) {
       const spaceGeom: FormConfigField = this.object.spaceGeom;
       return spaceGeom.embeddedFields.latitude;
     }
-    if (this.object && this.fieldObject.latitude) {
+    if (this.object && this.fieldObject && this.fieldObject.latitude) {
       return this.fieldObject.latitude;
     } else {
       return this.formService.getEmptyConfigField();
     }
   }
 
   get geometry(): FormConfigField {
+    if (this.fieldObject === undefined) {
+      return this.formService.getEmptyConfigField();
+    }
     const geo = this.fieldObject.geometry || this.formService.getEmptyConfigField();
     return geo;
   }
 
   get longitudeField(): FormConfigField {
-    if (this.object.isSpaceGeom) {
+    if (this.object.isSpaceGeom && this.object.isSpaceGeom.embeddedFields) {
```

Comment:
embeddedFields are prop of object not isSpaceGeom, that is boolean

Reply:
fixed